notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -54,12 +54,13 @@
   },
```

Comment:
<p>Should we clear cells output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>As I understood, cells output should be preserved for documentation building.</p>